protocol/messenger_communities.go

```diff
@@ -139,7 +142,7 @@ func (m *Messenger) JoinCommunity(ctx context.Context, communityID types.HexByte
 
 	communitySettings := communities.CommunitySettings{
 		CommunityID:                  communityID.String(),
-		HistoryArchiveSupportEnabled: true,
```

Comment:
Ah, this should've probably been `false` for now in the previous commit as well.